### DISCUSSION TITLE
订阅链接无法使用。

### DISCUSSION THREAD

**Discussion by Beyon-cmd** [jonssonyan/h-ui#101] [General]
配置好了，所有的一切,它都显示歇斯底里已经在运行中，但是订阅链接无法使用。v2ray显示![Screenshot_20241231_214717_com.v2ray.ang.jpg](https://github.com/user-attachments/assets/e5954a2f-0db3-4c5b-8033-15d0bc54294b)

**Comment by Beyon-cmd** [comment]
求作者帮忙解决一下。或者来个大佬解决一下。

**Comment by jonssonyan** [comment]
你这什么客户端？现在支持Shadworocket Clash v2rayN

**Comment by Beyon-cmd** [comment]
我使用的是v2ray,现在问题已经解决了。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Beyon-cmd** [comment]
但是速度非常慢。完全体现不出hy2的优点。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by jonssonyan** [reply]
不应该啊，看每个人实际体验吧

**Comment by Beyon-cmd** [comment]
获取Outlook for Android <https://aka.ms/AAb9ysg>
------------------------------
*发件人：* yu lixin < ***@***.*** >
*发送时间：* 2025 年 1 月 1 日星期三上午 8:14:06
*收件人：* jonssonyan/h- ui
***@***.***><回复
***@***.***>+ ;
jonssonyan/h-dog < ***@***.*** ***@***.***> >; >
***@***.***>抄送：作者 < ***@***.***
***@***.***>
>主题：回复：[jonssonyan/h-ui] ***@***.***
***@***.***>
<*作者*
*：*抄送>
***@***.***>
***@***.***>

但是速度非常慢。完全体现不出hy2的优点。

获取Outlook for Android <https://aka.ms/AAb9ysg>
------------------------------
*发件人：* yu lixin < ***@***.*** >
*发送时间：* 2025 年 1 月 1 日星期三上午 8:12:45
*收件人：* jonssonyan/h- ui
***@***.***><回复
***@***.***>+ ;
jonssonyan/h-ui < ***@***.*** >
*抄送：*作者 < ***@***.*** >
*主题：*回复：

我使用的是v2ray,现在问题已经解决了。

获取Outlook for Android <https://aka.ms/AAb9ysg>
------------------------------
*From:* jonssonyan ***@***.***>
*Sent:* Tuesday, December 31, 2024 10:52:42 PM
*To:* jonssonyan/h-ui ***@***.***>
*Cc:* Beyon-cmd ***@***.***>; Author ***@***.***>
*Subject:* Re: [jonssonyan/h-ui] 订阅链接无法使用。 (Discussion #101)


你这什么客户端？现在支持Shadworocket Clash v2rayN

—
直接回复此邮件、 在 GitHub 上查看
<https://github.com/jonssonyan/h-ui/discussions/101#discussioncomment-11706118>
或 取消订阅
<https://github.com/notifications/unsubscribe-auth/BJRHCS7OYEYNYYXQGQAQZQL2IKVTVAVCNFSM6AAAAABUNQHX72VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNZQGYYTCOA>
。
您收到此邮件是因为您是此主题的作者。消息 ID：< jonssonyan/ h - ui /repo-
***@***.*** >
